lib/strict.pm

```diff
@@ -59,14 +61,15 @@ sub bits {
 
 sub import {
     shift;
-    $^H |= @_ ? &bits : all_bits | all_explicit_bits;
+    $^H |= @_ ? &bits | $explicit_bits : all_bits | all_explicit_bits;
```

Comment:
`&bits | $explicit_bits` relies on left-to-right evaluation in `|` since `&bits` resets the value of `$explicit_bits`. Does perl guarantee left-to-right evaluation in non-shortcutting operators?